spec/spec_helper.rb

```diff
@@ -2,14 +2,35 @@
 
 require 'hyperclient'
 require 'webmock/rspec'
-
 require 'slack-ruby-bot-server/rspec'
 
 SlackRubyBotServer::Service.logger.level = Logger::WARN
 
-Mongo::Logger.logger.level = Logger::INFO
-Mongoid.load!(File.expand_path('../sample_app/config/mongoid.yml', __dir__), ENV['RACK_ENV'])
+files       = Dir[File.join(__dir__, 'support', '**/*.rb')]
+mongo_files = ["#{File.dirname(__FILE__)}/support/mongoid.rb"]
+
+if defined? Mongo
```

Comment:
Spearate into subfolders and load, eg. "spec/support/databases/mongoid.rb" and "pg.rb"